pkg/readiness/ready_tracker.go

```diff
@@ -46,6 +46,8 @@ import (
 
 var log = logf.Log.WithName("readiness-tracker")
 
+var crashOnFailureFetchingExpectations = flag.Bool("crash-on-failure-fetching-expectations", false, "Unless set (defaults to false), gatekeeper will ignore errors that occur when gathering expectations. This prevents bootstrapping errors from crashing Gatekeeper at the cost of increasing the risk Gatekeeper will under-enforce policy by serving before it has loaded in all policies. Enabling this will help prevent under-enforcement at the risk of crashing during startup for issues like network errors. Note that enabling this flag currently does not achieve the aforementioned effect since fetching expectations are set to retry until success so failures during fetching expectations currently do not occur.")
```

Comment:
nit: I think this can be shortened

Reply:
Do you mean the naming or the description or both?

Reply:
Description